src/hardware/driver/mod.rs

```diff
@@ -51,7 +50,7 @@ impl ChannelVariant {
     const R_OUT_LN: f32 = 40.0; // Low noise side output resistor
     const R_OUT_HP: f32 = 0.068; // High power side output resistor
     const DIVIDER_HP: f32 = 1. / 41.; // High power side DAC output divider ratio
-    fn transimpedance(&self) -> f32 {
+    fn scale(&self) -> f32 {
```

Comment:
Scale for what? Transimpedance of what? What's the difference? Please document and find unambiguous names.